FAQ.md

```diff
@@ -366,9 +366,9 @@ Unfortunately this cannot be fixed from NetGuard, and can be considered a shortc
 <a name="FAQ39"></a>
 **(39) Which usage data will be shared?**
 
-On Android version 5.0 or newer, the free version of NetGuard will share anonymized usage data to help improve how NetGuard protects you,
+On Android version 5.0 or newer, NetGuard will share anonymized usage data to help improve how NetGuard protects you,
 by understanding how end users work with NetGuard.
-This can optionally be disabled after purchasing one or more of the pro features.
+Users can optionally opt-out of sending this data on a per-app basis.
```

Comment:
"Sending usage data can optionally be disabled per application."

I try to prevent using the word user, because it sounds somewhat awkward to me.